subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/plugins/JacocoPlugin.java

```diff
@@ -247,4 +249,10 @@ public File call() {
             }
         });
     }
+
+    private void addDefaultCheckTask(final Test task) {
+        final JacocoCheck checkTask = project.getTasks().create("jacoco" + StringUtils.capitalise(task.getName()) + "Check", JacocoCheck.class);
```

Comment:
Same nitpick as elsewhere: `jacocoTestCheck` seems like a confusing name to me. What name would make it very obvious what it's purpose was?